server/api/user.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router()
+const { User, Order, Costume } = require('../db')
+
+router.get('/', async(req, res, next) => {
```

Comment:
These are **enormous security violations.** You are allowing anybody to fetch all your users information - **including passwords!** These routes need to be locked down and/or removed.